site/specs/numbers.json

```diff
@@ -1536,7 +1536,7 @@
           "/accounts"
         ],
         "description": "<p>Creates a bulk port-in order to be used as a template for a collection of child port-in orders.  The template values will be cascaded to child port-ins that result from decomposing a collection of Telephone Numbers that span carriers, RespOrgs, or have attributes that drive the decomposition into a number of individual port-in orders.\nUpon a successfully-submitted payload, the order will have a status of \"DRAFT\", denoting that further modification to the template is expected.  For example, the next step is to use the /tnList endpoint to add a collection of telephone numbers to the bulk port-in order.\nThe only valid value for the ProcessingStatus element in a POST is 'DRAFT', which is the default value.</p>\n",
-        "operationId": "POST /accounts/{accountId}/bulkPortins",
+        "operationId": "POST bulk port-in order",
```

Comment:
We don't need to leave the `method` in the friendly name.  The docs will put the method in there already.
![image](https://user-images.githubusercontent.com/2125458/158380167-4a9dfbbc-2439-41a4-b02f-660f09cc9015.png)